doc/install/index.rst

```diff
@@ -28,7 +28,8 @@ precompiled wheel for your OS and Python.
    The following backends work out of the box: Agg, ps, pdf, svg
 
    Python is typically shipped with tk bindings which are used by
-   TkAgg.
+   TkAgg. Notably, python-build-standalone -- used by ``uv``  -- does
```

Comment:
```suggestion
   TkAgg. Notably, python-build-standalone – used by ``uv`` – does
```

use actual endash because we turn off the auto conversion from -- to – because it breaks something else in the docs